src/critic_manager.cpp

```diff
@@ -49,8 +49,8 @@ std::string CriticManager::getFullName(const std::string & name)
   return "mppi::critics::" + name;
 }
 
-void CriticManager::evalTrajectoriesScores(
-  models::CriticFunctionData & data) const
```

Comment:
Personally, I would prefer something other than `score` since all of the critics' API uses `score` for the critic-level API name. Just for skimming purposes, I liked the `evalTrajectoryScores` API.

Reply:
It is not only trajectories scoring anymore, so i assumed we need more accurate naming

Reply:
What do you mean? The critics are scoring the trajectories -- I don't see anything else this function does

Reply:
Ah sure, trajectories is accurate . nvm. blunted a little

Reply:
brought back